mvg/mvg.py

```diff
@@ -257,17 +243,27 @@ def create_source(self, sid: str, meta: dict):
 
         meta : dict
             meta information
+
+        ignore_conflict : bool
```

Comment:
The word conflict is coming from the title of HTTP error 409. From user's point of view this error happens when the source or measurement already exists. What about calling it `ignore_existing` or similar to `mkdir` call it `exist_ok`.